src/cdsetool/download.py

```diff
@@ -67,27 +93,20 @@ def download_feature(
                 log.warning("Token signature expired, retrying..")
                 continue
             url = _follow_redirect(url, session)
-            name_dir_prefix = filename.replace(".zip", "____")
-            with (
-                session.get(url, stream=True) as response,
-                tempfile.TemporaryDirectory(
-                    prefix=name_dir_prefix, dir=temp_dir_usr
-                ) as temp_dir,
-            ):
+            with session.get(url, stream=True) as response:
```

Comment:
The temporary directory stuff is intricate and I have to run now, so I need to look at the consequences of removing that later.